server/routes/varyingLicences/handlers/caseload.ts

```diff
@@ -29,7 +29,7 @@ export default class CaseloadRoutes {
           crnNumber: c.deliusRecord.otherIds.crn,
           licenceType: licence.type,
           releaseDate: moment(c.nomisRecord.releaseDate, 'YYYY-MM-DD').format('DD MMM YYYY'),
-          licenceStatus: licence.status,
+          licenceStatus: licence.status === LicenceStatus.ACTIVE ? null : licence.status,
```

Comment:
Setting to null will prevent the statusBadge macro rendering the badge

Reply:
Could this check be placed within the Macro itself? The only issue I can see here is if something now or in the future references the licenceStatus property, downstream from the handler.